### PR TITLE
Add short help switch and better parameter names

### PR DESCRIPTION
**Pull request by khardix**
Fixes sclorg#53.

I added all the metavar names as they were before. The two exceptions are `--build` and `--download`, where the listing of accepted values seems more practical.

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -189,3 +189,15 @@ def test_simple_options(runner, recipe_arguments, option, value):
...
+
+@pytest.mark.parametrize('help_switch', ['--help', '-h'])
+def test_help_option_variants(runner, help_switch, recipe_arguments):
+    """Test that both short and long version of help switch works."""
+
+    # NOTE: recipe_arguments are necessary, both run.make_context and
+    # runner.invoke break without them for some reason
+    result = runner.invoke(run, [help_switch] + recipe_arguments)
+
+    print(result.output)
```
Comment (junaruga) [inline tests/test_cli.py]: This `print` is added intentionally?

Comment (khardix) [inline tests/test_cli.py]: Yes, for debugging purposes. If the tests fails, the help message is shown as captured stdout.

Comment (junaruga) [inline tests/test_cli.py]: OK.
You can see when the tests are success, the 2 help messages are printed on result of the unit tests.
Is this also showed intentionally?
https://travis-ci.org/sclorg/rpm-list-builder/jobs/251988985

Comment (khardix) [inline tests/test_cli.py]: Ah, apparently with `pytest -v` (verbose mode), the stdout is showed every time – which is unnecessary clutter. In this case, I will replace the print with something else.

Comment (hroncok) [inline tests/test_cli.py]: It should not work that way. That's actually weird.

Comment (khardix) [inline tests/test_cli.py]: @hroncok My bad, it is not the `-v` switch, it is the `-s` switch 😊

Comment (hroncok) [inline tests/test_cli.py]: And that's how it's supposed to work. Keep the print. Get rid of the -s switch if used.

**Comment by khardix** [comment]
@junaruga I merged your PR 🎉 , can you approve the changes?

**Comment by junaruga** [comment]
Yes, I approved! :)

**Pull request merged**